### ISSUE TITLE
disabledFetchingCountry not working

### ISSUE THREAD

**Issue by diadal** [EducationLink/vue-tel-input#37]
is not working am still getting hit to this url `https://ipinfo.io/json?callback=__jp0`

```
<vue-tel-input v-model="phone" 
@onInput="onInput"
:disabledFetchingCountry="true"
:placeholder="'Enter Your Phone Number'"
 >
</vue-tel-input>
```

**Comment by fabiosal** [comment]
added pull request related to this issue #40

**Comment by iamstevendao** [comment]
Thanks, @fabiosal. This should be fixed in v2.0.6
